db/schema.rb

```diff
@@ -107,6 +107,7 @@
     t.datetime "created_at"
     t.datetime "updated_at"
     t.string   "ancestry"
+    t.string   "contact_finder_uuid"
```

Comment:
Can we name this differently, maybe `contact_index_content_id`.  The fact that it contains a UUID is an implementation detail.  We're referring to this as the `content_id` everywhere else in the stack.